modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc

```diff
@@ -77,6 +78,12 @@ class NDB_Menu_timepoint_list extends NDB_Menu
                 $this->tpl_data['timePoints'][$x]['currentStatus'] = $timePoint->getCurrentStatus();
                 $this->tpl_data['timePoints'][$x]['currentDate'] = $timePoint->getCurrentDate();
                 
+
+                // get Center Name
+		$centerID = $timePoint->getData('CenterID');
+		$center = $DB->pselectRow("SELECT CenterID as ID, PSCArea as Name FROM psc WHERE CenterID =:cid", array('cid' => $centerID));
```

Comment:
@driusan 
Is the CenterName the Name column or the PSCArea column in the psc table?